.gitignore

```diff
@@ -1,5 +1,6 @@
 # Addon input files
```

Comment:
Can you tack on 
```
build/ 
dist/ 
wow_addon_updater.egg-info/ 
```
at the end? Like to keep it clean, and avoids people committing some junk accidentally.

Reply:
:+1: